src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -157,20 +194,71 @@ class WorkflowView extends Component {
   }
 
   renderDetail = () => {
-    const { selectedTab, wdl } = this.state
+    const { selectedTab, wdl, inputsOutputs, config, modifiedAttributes } = this.state
 
-    if (selectedTab === 'WDL') {
-      return wdl ?
-        div({
-          style: {
-            flex: '1 1 auto', overflowY: 'auto', maxHeight: 500,
-            margin: '0 3rem', padding: '0.5rem', backgroundColor: 'white',
-            border: Style.standardLine, borderTop: 'unset'
+    if (selectedTab === 'WDL' && wdl) {
+      return div({
+        style: {
+          flex: '1 1 auto', overflowY: 'auto', maxHeight: 500,
+          margin: `0 ${sideMargin}`, padding: '0.5rem', backgroundColor: 'white',
+          border: Style.standardLine, borderTop: 'unset'
+        }
+      }, [h(WDLViewer, { wdl, readOnly: true })])
+    } else if (selectedTab !== 'WDL' && inputsOutputs) {
+      const key = selectedTab.toLowerCase() // 'inputs' or 'outputs'
+
+      return div({ style: { margin: `0 ${sideMargin}` } },
+        [h(DataTable, {
+          dataSource: inputsOutputs[key],
+          allowPagination: false,
+          customComponents: components.fullWidthTable,
+          tableProps: {
+            showHeader: false, scroll: { y: 450 },
+            columns: [
+              {
+                key: 'task-name', width: 350,
+                render: ({ task }) =>
+                  div({
+                    style: {
+                      fontWeight: 500,
+                      overflow: 'hidden', textOverflow: 'ellipsis'
+                    }
+                  }, task)
+              },
+              {
+                key: 'variable', width: 360,
+                render: ({ variable, optional }) =>
+                  styleForOptional(optional, variable)
+              },
+              {
+                key: 'type', width: 160,
+                render: ({ inputType, outputType, optional }) =>
+                  styleForOptional(optional, inputType || outputType)
+              },
+              {
+                key: 'attribute', width: '100%',
+                render: ({ name, optional }) => {
+                  let value = modifiedAttributes[key][name]
+                  if (value === undefined) {
+                    value = config[key][name]
+                  }
+                  return textInput({
+                    name, value,
+                    type: 'search',
+                    placeholder: optional ? 'Optional' : 'Required',
+                    onChange: e => {
+                      modifiedAttributes[key][name] = e.target.value
+                      this.setState({ modifiedAttributes, modified: true, untouched: false })
+                    },
+                    style: { margin: '-10px 0 -6px' }
```

Comment:
This was my way of keeping the input from blowing up the row height. There's probably a better way.